firmware/logfs/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+message("")
+message("⚙️ Configuring LogFS")
```

Comment:
Legit to leave this here - just checking that this isn't changes from another PR leaking into this one - if it isn't ignore this comment 😅

Reply:
oh i was just debugging